client/src/stylesheets/login.css

```diff
@@ -1,9 +1,33 @@
-.loginForm {
+.background {
+  background-image: url('./images/luca-bravo-ESkw2ayO2As-unsplash.jpg');
+  background-repeat: no-repeat;
+  background-size: 100%;
```

Comment:
It might be nice to add some comments in here describing the intent of the various css styles. I know this was something that took a lot of fiddling to get right and that info would be useful for future developers when they come back to this